content/well-architected-framework/docs/docs/automate-and-define-processes/create-high-performing-teams/collaboration.mdx

```diff
@@ -0,0 +1,30 @@
+---
+page_title: Collaboration
+description: Learn methods for team collaboration.
+---
+
+# Collaboration
+
+Collaboration amongst team members is important for the quality and quantity of work produced. Tools and processes can enable team collaboration.
+Git and peer reviews
+
+## Git and peer reviews
+
+Version control tools like Git are key for team coding projects. Many teams use online platforms such as GitHub, GitLab, or BitBucket to manage their Git repositories. These services make it easy to share code and get feedback from colleagues. A main feature is the pull request (sometimes called a merge request), which lets developers propose changes to the codebase. When you create a pull request, team members can review your code, leave comments, suggest improvements, and test the changes before they're added to the main project. This process helps maintain code quality, spreads knowledge within the team, and keeps a record of how the code has evolved over time.
+Planning as a team
+
+## Planning as a team
+
+Proper planning is important for developing and delivering your application and new features. If you are planning in a silo, where only management plans development cycles, you can miss important factors that affect the development of your application. When you plan as a team, you get more information to make accurate timelines on projects, along with everyone being on the same page.
+
+It is important to capture the work you plan in the planning session. You can track work by using project management software such as JIRA, Asana, and Trello. These project management tools help ensure the team is on track and allow team members to see what others are working on. Work transparency can increase team collaboration by increasing shared knowledge. If an individual has experience with a tool or process that another team member is lacking, they can lend a hand with that team member's task.
+
+External resources:
+
+- [About pull requests ](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#:~:text=A%20pull%20request%20is%20a,changes%20into%20the%20main%20codebase.) - GitHub
+- [Git documentation](https://git-scm.com/doc)
+- Project management tools: [JIRA](https://www.atlassian.com/software/jira), [Asana](https://asana.com/), [Trello](https://trello.com/)
+
+## Next steps
+
+In this section of how to [create high-performing teams](/well-architected-framework/automate-and-define-processes/create-high-performing-teams/overview), you learned about methods to increase team collaboration. Creating high performing teams is part of the Automate and define processes pillar.
```

Comment:
let's link the "automate and define processes" pillar

Reply:
That's the plan, we just don't have a automate and define processes pillar overview yet. Once that is made, I will go back and link all the docs to them.